src/data/configs/config.json

```diff
@@ -190,7 +208,8 @@
       "Orikan the Diviner",
       "The Silent King",
       "Trazyn the Infinite"
-    ]
+    ],
+    "dedicated-transport": ["Sagitaur"]
```

Comment:
I think this should be 'Ghost Ark'?

Reply:
There are a few missing as well, let me fix it

Reply:
Please resolve this if you are happy with the commit